sale_properties_dynamic_fields/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+==============================
+Sale properties dynamic fields
+==============================
+
+This module allows to dynamically draw properties groups on sale order lines.
```

Comment:
the description can be updated, so can the displayed strings.

I understand if you don't want to brake the API of the addon. In that case can you add a docstring of draw_dynamically and a comment saying "TODO: rename method when porting to Odoo v9"